website/pages/docs/providers/vmware/vagrant-vmware-utility.mdx

```diff
@@ -116,3 +116,60 @@ $ sudo /etc/init.d/vagrant-vmware-utility start
 ```shell-session
 $ sudo sv start vagrant-vmware-utility
 ```
+
+## Utility Service Configuration
+
+When installing the Vagrant VMware utility service, a configuration file is generated
+that is used when the process is started. On windows, this can be found at:
+
+```text
+C:\ProgramData\HashiCorp\vagrant-vmware-desktop\config\service.hcl
+```
+
+On POSIX-style systems, it can be found at:
+
+```text
+/opt/vagrant-vmware-desktop/config/service.hcl
+```
+
+The configuration file uses the [HCL](https://github.com/hashicorp/hcl) configuration
```

Comment:
It would be neat to add a bit about how a user can generate a config file using the api. eg. `./vagrant-vmware-utility service install -h`